airbyte-integrations/connectors/source-mixpanel/acceptance-test-config.yml

```diff
@@ -1,6 +1,9 @@
 # See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-mixpanel:dev
+custom_environment_variables:
+  SLEEP_BETWEEN_CHECK_AND_DISCOVER_REQUESTS: true
+  ALIGN_DATE_RANGE_TO_LAST_N_DAYS: true
```

Comment:
We already have `AVAILABLE_TESTING_RANGE_DAYS` defined as a constant and only used during testing.

Perhaps we just update it to be an env var `AVAILABLE_TESTING_RANGE_DAYS` and set it to 10 here

Then in the code we can write

```py
    @wraps(func)
    def wrapper(self, config):
        config = func(self, config)

        # Some comment explaining why we have to do this for testing...
        AVAILABLE_TESTING_RANGE_DAYS = os.environ.get("ALIGN_DATE_RANGE_TO_LAST_N_DAYS", 0)

        if AVAILABLE_TESTING_RANGE_DAYS:
            logger = logging.getLogger("airbyte")
            logger.info("SOURCE IN TESTING MODE, DO NOT USE IN PRODUCTION!")
            # Take time range in only 1 month
            if (config["end_date"] - config["start_date"]).days > AVAILABLE_TESTING_RANGE_DAYS:
                config["start_date"] = config["end_date"].subtract(days=AVAILABLE_TESTING_RANGE_DAYS)
        return config
```